scipy/special/orthogonal.py

```diff
@@ -1143,6 +1143,25 @@ def hermite(n, monic=False):
     The polynomials :math:`H_n` are orthogonal over :math:`(-\infty,
     \infty)` with weight function :math:`e^{-x^2}`.
 
+    Examples
+    --------
+    >>> from scipy import special
+    >>> import matplotlib.pyplot as plt
+    >>> import numpy as np
+
+    >>> p_monic = special.hermite(3, monic=True)
+    >>> p_monic
+    poly1d([ 1. ,  0. , -1.5,  0. ])
+    >>> p_monic(1) # Evaluate to a point
```

Comment:
Nitpick: we don't really need this comment and the other comments below.  They don't add information that is not already clear from the statement itself.

Reply:
Removed comments